view/Requirements.php

```diff
@@ -1482,20 +1515,20 @@ public function getCombinedFilesEnabled() {
 	 * any of these files have changed.
 	 *
 	 * @param array $fileList List of files
-	 * @return int Last modified timestamp of these files
+	 * @return string SHA1 bashed file hash
 	 */
-	protected function getEntropyOfFiles($fileList) {
-		// file exists, check modification date of every contained file
+	protected function hashOfFiles($fileList) {
+		// Get hash based on hash of each file
 		$base = Director::baseFolder() . '/';
-		$srcLastMod = 0;
+		$hash = '';
 		foreach($fileList as $file) {
 			if(file_exists($base . $file)) {
-				$srcLastMod = max(filemtime($base . $file), $srcLastMod);
+				$hash .= sha1_file($base . $file);
 			} else {
 				throw new InvalidArgumentException("Combined file {$file} does not exist");
```

Comment:
His use case was `tiny_mce_gzip.php?m=1434761806&js=1&plugins=table,emotions,paste,media,fullscreen...`, but we don't run that through `combine_files()` anyway. Cleared up that misunderstanding with @caffeineinc, we think there's no need to support these "dynamically included files" in the new API